common/g_mapinfo.cpp

```diff
@@ -785,31 +785,78 @@ void MIType_MapName(OScanner& os, bool newStyleMapInfo, void* data, unsigned int
 {
 	ParseMapInfoHelper<std::string>(os, newStyleMapInfo);
 
-	if (IsIdentifier(os))
+	if (os.compareTokenNoCase("EndPic"))
 	{
-		if (os.compareTokenNoCase("EndPic"))
-		{
 			// todo
 			if (newStyleMapInfo)
-				MustGetStringName(os, ",");
+			MustGetStringName(os, ",");
 
 			os.mustScan();
-		}
-		else if (os.compareTokenNoCase("EndSequence"))
-		{
+	}
+	else if (os.compareTokenNoCase("EndSequence"))
+	{
 			// todo
 			if (newStyleMapInfo)
-				MustGetStringName(os, ",");
+			MustGetStringName(os, ",");
```

Comment:
This is a very suspect unbraced block spacing change.

Reply:
Fixed, thanks for double checking that, that was not intentional